src/eko/io/runcards.py

```diff
@@ -48,10 +46,16 @@ class TheoryCard(DictLike):
     xif: float
     """Ratio between factorization scale and process scale."""
     n3lo_ad_variation: N3LOAdVariation
-    """|N3LO| anomalous dimension variation: ``(gg_var, gq_var, qg_var, qq_var)``."""
+    """|N3LO| anomalous dimension variation: ``(gg_var, gq_var, qg_var,
+    qq_var)``."""
     matching_order: Optional[Order] = None
     """Matching conditions perturbative order tuple, ``(QCD, QED)``."""
 
+    def __post_init__(self):
+        """Enforce defaults."""
+        if self.matching_order is None:
+            self.matching_order = self.order
```

Comment:
This is the only relevant part. The additional noise has been added by the formatter (I usually try to avoid this, but I authored the PR too quickly...)